examples/example-gateway/gateway.json

```diff
@@ -1,5 +1,6 @@
 {
 	"gatewayName": "example-gateway",
+	"packageRoot": "github.com/uber/zanzibar/example-gateway",
```

Comment:
Because I am injecting a new source of truth. Changing all of the code to use this new source of truth is what I'm trying to do but I can't do it without this first. And yes this is the wrong value, I didn't pull that fix from the right commit (and we wouldn't notice because it isn't used to generate anything yet).